packages/js/product-editor/src/blocks/product-fields/description/components/with-full-editor-toolbar-button.tsx

```diff
@@ -23,11 +23,6 @@ const wooBlockwithFullEditorToolbarButton =
 					return <BlockEdit { ...props } />;
 				}
 
-				// Only add the `Full editor` button when the block is selected
-				if ( ! props?.isSelected ) {
-					return <BlockEdit { ...props } />;
-				}
```

Comment:
I am not sure why this condition got added in the first place, do you know @nathanss ? it doesn't seemed to affect anything on the surface, but I wonder if we are missing an edge case?
cc: @retrofox  who implemented it.

Instead of fully removing the condition could we also just move the condition inline. So the return method looks like this:
```
<>
	{ props?.isSelected && (
		<BlockControls { ...blockControlProps }>
			<FullEditorToolbarButton />
		</BlockControls>
	) }
	<BlockEdit { ...props } />
</>
```

Reply:
I don't recall it. It was mostly part of a refactoring process, moving the Modal component out of the block's edit function component. I'm going to take a look.
Anyway, It seems to be pretty safe to remove it.

> Instead of fully removing the condition, could we also just move the condition inline? So the return method looks like this:

It seems almost the same, doesn't it? It ends up being a matter of code clarity or legibility.

Reply:
In that case I think its safe to remove it, I will approve the PR.

Reply:
I'm okay changing the code, too, just in case :-)